lib/std/os/bits/openbsd.zig

```diff
@@ -32,11 +32,14 @@ pub const Kevent = extern struct {
     udata: usize,
 };
 
+const Elf_Addr = if (@sizeOf(usize) == 8) std.elf.Elf64_Addr else std.elf.Elf32_Addr;
```

Comment:
perhaps: 
```zig
const Elf_Addr = switch(@sizeOf(usize)) {
  4 => std.elf.Elf32_Addr,
  8 => std.elf.Elf64_Addr,
  else => @compileError("unknown pointer size"),
};
```

Reply:
This may be useful in `elf.zig` as `Addr`, we have a few other target-dependent definitions.